docs/content/doc/advanced/customizing-gitea.en-us.md

```diff
@@ -139,6 +139,39 @@ you would need to remove `{{if .RequireHighlightJS}}` and `{{end}}`.
 
 Mermaid will detect and use tags with `class="language-mermaid"`.
 
+#### Example: PlantUML
+
+You can add [PlantUML](https://plantuml.com/) support to Gitea's markdown by using a PlantUML server.
+The data is encoded and sent to the PlantUML server which generates the picture. There is an online 
+demo server at http://www.plantuml.com/plantuml, but if you (or your users) have sensitive data you 
+can set up your own [PlantUML server](https://plantuml.com/server) instead. To set up PlantUML rendering
+copy javascript files from https://gitea.com/davidsvantesson/plantuml-code-highlight and put in your
+`custom/public` folder. Then add the following to `custom/footer.tmpl`:
+
```

Comment:
```suggestion

For the actual location of the `custom` directory, check the "Configuration" tab in the "Site Administration" page.

```

I think we should start introducing this kind of hints in the docs when possible. This is only a suggestion, of course.

Reply:
I added information on top of the page instead. People have to learn reading the full page 😁